runner/ansible/vars/aws/10-default.yml

```diff
@@ -0,0 +1,27 @@
+---
+
+expected_core:
+  "1.1.1": "30000"
+  "1.1.2": "36000"
+  "1.1.3": "" # unkown value, default may apply
```

Comment:
Typo here `unkown` -> `unknown`.
Anyway, what means `default may apply`?
If we leave this blank, what will happen is that the check will use an empty value in the comparison, which will produce a failure in the checks result most probably. I don't think this is what we want, right?
I see 2 options. We put any value (what we have in Azure for example), or at least we remove the `default may apply` comment, which would confuse someone who reads this